examples/tinywl/StackWorkspace.qml

```diff
@@ -81,14 +94,41 @@ Item {
 
         // TODO: Support server decoration
         XdgSurface {
-            id: surface
+            id: toplevelSurfaceItem
             property var doDestroy: helper.doDestroy
             property var cancelMinimize: helper.cancelMinimize
+            property int outputCounter: 0
+
+            OutputLayoutItem {
```

Comment:
No, there is a `OutputLayoutItem` in `XdgSurface.qml`

Reply:
Removed in XdgSurface.qml